invenio_app_rdm/theme/templates/semantic-ui/invenio_requests/community-submission/index.html

```diff
@@ -0,0 +1,58 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2020 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_requests/details/index.html" %}
+
+{% block javascript %}
+  {{ super() }}
+  <script>
+    $('.menu .item').tab();
+  </script>
+{% endblock %}
+
+{% block request_topic %}
+  <h1 class="ui header">
+    <div class="ui label green">New submission</div>
+    {{ request.topic.metadata.title }}</h1><br/>
+  <div class="ui horizontal list">
+    {% for item in request.topic.metadata.creators %}
+      <div class="item">
+        <div class="content"><i class="user icon"></i>
+          {{ item.person_or_org.name }}</div>
+      </div>
+    {% endfor %}
```

Comment:
Is it possible to have many authors? 
If yes, maybe we should add an expand button or limit the displayed creators?

Reply:
See similar issue: https://github.com/inveniosoftware/invenio-communities/issues/401

Reply:
I will create an issue for this, because the PR is growing too big
btw, I think it should be initially fixed in the record_ui templates. I will suggest that in the ticket